lib/caseflow/schema/appeals.json

```diff
@@ -93,7 +93,7 @@
                 }
               },
               "aod": {
-                "type": "boolean"
+                "type": ["boolean", "null"]
```

Comment:
This can be `null` because of [this line in Caseflow](https://github.com/department-of-veterans-affairs/caseflow/blob/master/app/models/appeal.rb#L628), which is invoked [here in the controller we call](https://github.com/department-of-veterans-affairs/caseflow/blob/master/app/controllers/api/v2/appeals_controller.rb#L29). The fallback using `||` does not work the way they intended - if the result of the fallback is `nil`, this expression will return `nil`, not `false`.

It would be nice to fix this at the source but I think we've had trouble making PRs and getting them reviewed there before, so I figured this would be acceptable